day_6/exercises/burrito.rb

```diff
@@ -5,15 +5,40 @@
 # 3. change_protein
 
 class Burrito
-  attr_reader :protein, :base, :toppings
-  def initialize(protein, base, toppings)
-    @protein  = protein
-    @base     = base
-    @toppings = toppings
+  attr_accessor :protein, :base, :toppings
+
+  def initialize(protein: protein, base: base, toppings: toppings)
+      @protein  = protein
+      @base     = base
+      @toppings = toppings
+        puts "-" * 10
+    puts "These ingredients are available!"
+        puts "-" * 10
+  end
+
+  def add_topping(topping)
+    @toppings << topping
+    @toppings
+
+  end
+
+  def remove_topping(topping)
+    index = @toppings.index(topping)
+    @toppings.delete_at(index)
```

Comment:
Nice job on this exercise Alex! As you may know, in ruby the last evaluated statement is returned from a method. Sometimes this is important if we are using that return value somewhere else in the program, but for the purposes of this exercise, once we change the @toppings array, we're not concerned with what happens after that. We also don't need to `p` each method call. We can just call the method and our code will run. Then you could `p` the dinner.toppings to see how the value changes after each method runs.